sdk/sui-owned-object-pools/.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
same thing as the eslintrc, we have a root prettier config shared by all other packages